modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -47,6 +47,14 @@ Default: `log`.
 
 When the user runs the tool with the wrapper script `must-gather` is supplied by the script itself and the user must not specify it.
 
+| `offlined-cpu-count`
+a| Number of offlined CPUs.
```

Comment:
maybe it worth to explain who is calculated?
To calculate offlined cpus we take into account the following input parameters:

    offlined-cpu-count : the number of cpus the user wants to set offline.
    disable-ht: Marks if the user want to disable logical processor siblings to disable hyperthreading
    power-consumption-mode: values: “default”, “low-latency”, “ultra-low-latency”

https://github.com/openshift/cluster-node-tuning-operator/pull/354#issuecomment-1129124730

Reply:
Hi @marioferh I found it difficult to get that into the docs without getting into the weeds I hope that is OK. I did note the error messages that will be printed.

Reply:
yep sure, the docs are well explained. It was just to go a little further.